### PR TITLE
landing page screen

### PR DESCRIPTION
**Pull request by TusharJade**
added landing page with responsiveness and with dark mode

### CODE REVIEW

**Review by harshitpaliwal95 — APPROVED** [review]
good file folder structure 
- ui looks great

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,29 @@
...
+} from "./pages/index";
 
 function App() {
-  return <div className="App">Hi</div>;
+  return (
+    <>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<LandingPage />} />
+        <Route path="/tasks" element={<CreateTaskPage />} />
```
Comment (bhavyaj12) [inline src/App.js:19]: Add 404 page too

File: src/components/Footer/Footer.css [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+}
+
+.footer-text{
+    font-size: 16px;
+    font-weight: 600;
+    margin: 10px 0 8px 0;
+}
+
+.code-icn{
+    color: #aba8a8;
```
Comment (bhavyaj12) [inline src/components/Footer/Footer.css:17]: Try to add this as global color variable

File: src/components/Navbar/Navbar.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    justify-content: space-between;
+    position: fixed;
+    z-index: 1;
+    top: 0;
+    background-color: var(--nav-bg);
+    box-shadow: var(--nav-box-shadow);
+    transition: 0.8s;
+}
+
+.kiazen-container{
```
Comment (bhavyaj12) [inline src/components/Navbar/Navbar.css]: Kiazen spelling

File: src/components/Navbar/Navbar.css [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+.theme-icn{
+    font-size: 1.25rem;
+    cursor: pointer;
+    transition: color 0.3s ease-in-out;
+    color: var(--nav-text);
+}
+
+.theme-icn:hover{
+    color: #664EFF;
```
Comment (bhavyaj12) [inline src/components/Navbar/Navbar.css]: Since this is being used multiple times add as color variable

File: src/App.js [hunk 5]
```diff
@@ -1,7 +1,29 @@
 import "./App.css";
+import Mockman from "mockman-js";
```
Comment (bhavyaj12) [inline src/App.js]: Try to follow the convention of global imports first then local imports

**Review by bhavyaj12 — APPROVED** [review]
Code is easy to understand and well structured.
Love the UI and smooth dark mode application 
Functionality works well.

File: src/App.js (hunk 1 above)
Comment (TusharJade) [reply src/App.js:19]: yes forgot to add will add it now

File: src/components/Footer/Footer.css (hunk 2 above)
Comment (TusharJade) [reply src/components/Footer/Footer.css:17]: yes will add it to global 👍

File: src/components/Navbar/Navbar.css (hunk 3 above)
Comment (TusharJade) [reply src/components/Navbar/Navbar.css]: opps, will fix that too..

File: src/components/Navbar/Navbar.css (hunk 4 above)
Comment (TusharJade) [reply src/components/Navbar/Navbar.css]: yes will do fix it

File: src/App.js (hunk 5 above)
Comment (TusharJade) [reply src/App.js]: didnt knew about this thanks from now will follow that 👍

**Pull request merged**